program/src/main/QuestionSet.vala

```diff
@@ -38,8 +48,21 @@ public class QuestionSet : Object{
         return rubricPool.get(ID);
     }
 
-    public void resizeBounds(double[] d){
-        
+    //There's probably a better way to do this but I'm new to vala
+    public bool bounds_is_null() { 
+        if (bounds[0] == 0 && bounds[1] == 0 && bounds[2] == 0 && bounds[3] == 0) { 
+            return true;
+        } else { 
+            return false;
+        } 
+    }
+
+    public void set_points(double points) { 
+        this.totalPoints = points;
+    }
+
+    public void set_bounds(double[] d){
```

Comment:
why name the variable d?

Reply:
I think that was a 4am spaghetti code monkey brain moment that somehow made a lot of sense when I wrote it at the time. I changed it to 'bounds' now, thanks for pointing that out!